tests/bitcoin/lock/LockMonitor.spec.ts

```diff
@@ -0,0 +1,713 @@
+import BitcoinClient from '../../../lib/bitcoin/BitcoinClient';
+import BitcoinError from '../../../lib/bitcoin/BitcoinError';
+import BitcoinLockTransactionModel from '../../../lib/bitcoin/models/BitcoinLockTransactionModel';
+import BitcoinTransactionModel from '../../../lib/bitcoin/models/BitcoinTransactionModel';
+import ErrorCode from '../../../lib/bitcoin/ErrorCode';
+import JasmineSidetreeErrorValidator from '../../JasmineSidetreeErrorValidator';
+import LockIdentifier from '../../../lib/bitcoin/models/LockIdentifierModel';
+import LockIdentifierSerializer from '../../../lib/bitcoin/lock/LockIdentifierSerializer';
+import LockMonitor from '../../../lib/bitcoin/lock/LockMonitor';
+import LockTransactionModel from '../../../lib/bitcoin/models/LockTransactionModel';
+import LockTransactionType from '../../../lib/bitcoin/enums/LockTransactionType';
+import MongoDbLockTransactionStore from '../../../lib/bitcoin/lock/MongoDbLockTransactionStore';
+import ValueTimeLockModel from '../../../lib/common/models/ValueTimeLockModel';
+
+function createLockInformation (latestSavedLockInfo: LockTransactionModel | undefined, currentValueTimeLock: ValueTimeLockModel | undefined) {

```

Comment:
Nit: for consistency, we have been using the abbreviated form throughout.
```suggestion
function createLockInfo (latestSavedLockInfo: LockTransactionModel | undefined, currentValueTimeLock: ValueTimeLockModel | undefined) {

```